templates/prescription/general_edit.html

```diff
@@ -4,21 +4,42 @@
 
 <link rel="stylesheet" href="{$CSS_HEADER}" type="text/css">
 <link rel="stylesheet" href="{$WEBROOT}/interface/themes/jquery.autocomplete.css" type="text/css">
+<link rel="stylesheet" href="{$WEBROOT}/library/js/jquery-ui.min.css" type="text/css" /> 
```

Comment:
Why do you need jquery-ui  in this script (I couldn't find anything you added that needs this)?